client/src/App.js

```diff
@@ -0,0 +1,86 @@
+import './styles.css';
+import { useData } from './context/DataContext';
+import { Routes, Route } from 'react-router-dom';
+import { useEffect } from 'react';
+import { Toast } from './components/Toast/Toast';
+import { Header } from './components/Header/Header';
+import { Explore } from './pages/Explore/Explore';
+import { VideoDetail } from './pages/VideoDetail/VideoDetail';
+import { LikedVideos } from './pages/LikedVideos/LikedVideos';
+import { History } from './pages/History/History';
+import { Playlists } from './pages/Playlists/Playlists';
+import { WatchLater } from './pages/WatchLater/WatchLater';
+import { Library } from './pages/Library/Library';
+import { handleFetchVideos } from './utils/requests';
+
+function App() {
+   const { dispatch } = useData();
+
+   useEffect(() => {
+      (async () => {
+         try {
+            const {
+               data: { videosLiked },
+            } = await handleFetchVideos(
+               'https://driftview-backend.duhbhavesh.repl.co/liked',
+            );
+            dispatch({ type: 'SET_VIDEOS_LIKED', payload: videosLiked });
+         } catch (err) {
+            console.log(err);
+         }
+      })();
+   }, [dispatch]);
+
+   useEffect(() => {
```

Comment:
rather than using useEffect for each call, you can club them and do inside a single useEffect........plz do let me know if this approach is better one!